src/styleguide/_patterns/04-templates/01-careers.mustache

```diff
@@ -1,10 +1,22 @@
 {{> organisms-page-header }}
 <main>
   <h1>{{title}}</h1>
+  <p>
+    <a href="#the-hiring-process">The hiring process</a>
+  </p>
 
   {{#locations}}
     {{> organisms-job-list }}
   {{/locations}}
 
+  <section>
+    <a name="the-hiring-process"></a>
```

Comment:
Could the `name` not be added to another element rather than having an empty anchor?